### ISSUE TITLE
Downgrade chaining

### ISSUE THREAD

**Issue by artehe** [stayintarkov/SIT.Manager.Avalonia#105]
One for the future I think but I've seen Ploot mention it before as well, we could work out additional available versions with the new install process by evaluating available downgrade patchers to see if there are routes to other versions by chaining downgrade patchers together

**Comment by Plootie** [comment]
Yeah I wanted to reimplement this at some point. It was removed because there was problems with the chaining logic when upgraders were also available. Was thinking maybe something _**fancy**_ like a rapidly exploring random tree or just something boring like looking for the patcher that gets us closest to the target version, rinse and repeat until one is found (or not)
